project_scrum/models/project_task.py

```diff
@@ -0,0 +1,156 @@
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_risk_field(self):
+        return [
+            ('0', '0'),
+            ('1', '1'),
+            ('2', '2'),
+            ('3', '3'),
+            ('5', '5'),
+            ('8', '8'),
+            ('13', '13'),
+            ('20', '20'),
+            ('40', '40'),
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_kano_field(self):
+        return [
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'),
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'),
+            ('not_set', 'Not Set'),
+        ]
+
+    user_id = fields.Many2one(
+        'res.users',
+        'Assigned to',
+        index=True,
+        track_visibility='onchange',
+    )
+    actor_ids = fields.Many2many(
+        comodel_name='project.scrum.actors',
+        string='Actor',
+    )
+    sprint_id = fields.Many2one(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    us_id = fields.Many2one(
+        comodel_name='project.scrum.us',
+        string='User Stories',
+        index=True,
+    )
+    date_start = fields.Datetime(
+        string='Starting Date',
+        required=False,
+    )
+    date_end = fields.Datetime(string='Ending Date')
+    use_scrum = fields.Boolean(related='project_id.use_scrum')
+    description = fields.Html('Description')
+    current_sprint = fields.Boolean(
+        compute='_compute_current_sprint',
+        string='Current Sprint',
+        search='_search_current_sprint',
+    )
+    moscow = fields.Selection('_get_moscow_field')
+    value = fields.Selection('_get_value_field')
+    risk = fields.Selection('_get_risk_field')
+    kano = fields.Selection('_get_kano_field')
+    color = fields.Integer(related='project_id.color')
+    reference = fields.Char(
+        'Number',
+        index=True,
+        readonly=True,
+        copy=False,
+        default=':',
+    )
+
+    @api.depends('sprint_id')
+    def _compute_current_sprint(self):
+        self.ensure_one()
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(
+            self.project_id.id)
+        if sprint:
+            self.current_sprint = sprint.id == self.sprint_id.id
+        else:
+            self.current_sprint = False
+
+    def _search_current_sprint(self, operator, value):
+        project_id = self.env.context.get('default_project_id', None)
+        sprint = self.env['project.scrum.sprint'].get_current_sprint(
+            project_id)
+        return [('sprint_id', '=', sprint and sprint.id or None)]
+
+    @api.multi
+    def write(self, vals):
+        if vals.get('stage_id') == self.env.ref(
+                'project.project_stage_data_2').id:
+            vals['date_end'] = fields.date.context_today()
```

Comment:
this looks more like something we want in https://github.com/OCA/project/blob/10.0/project_stage_state and have this module depend on it